src/rm.js

```diff
@@ -149,6 +149,11 @@ function _rm(options, files) {
     if (stats.isDirectory() && options.recursive) {
       rmdirSyncRecursive(file, options.force);
     }
+
+    // Path is a symbolic link
+    if (stats.isSymbolicLink()) {
```

Comment:
We can add support for the others later. In this PR, we need to make sure that we have an `else` case that performs whatever would have been done before (assuming it was reasonable behavior) to avoid a breaking change.